bindings/python/tests/test_reader_datetime.py

```diff
@@ -71,7 +71,7 @@ def test_read_with_timestamps(self):
 
         mock.add_data(numpy.arange(10))
 
-        values, domain, status = reader.read_with_timestamps(10)
+        values, domain, status = reader.read_with_timestamps(10, 0, True)
```

Comment:
These are just tests, but I'd maybe write these in the format of:

```
values, domain, status = reader.read_with_timestamps(10, returnStatus=true)
```

Reply:
I’ve tested it with examples from **examples/python/guide**. They are unchanged and work ok